packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -61,7 +61,7 @@ class ThemeData {
     primarySwatch ??= Colors.blue;
     primaryColor ??= isDark ? Colors.grey[900] : primarySwatch[500];
     primaryColorBrightness ??= ThemeBrightness.dark;
-    accentColor ??= primarySwatch[500];
+    accentColor ??= isDark ? Colors.tealAccent[200] : primarySwatch[500];
```

Comment:
I picked the color that matched the accent color used by MD for the dark theme date picker examples: https://www.google.com/design/spec/components/pickers.html#pickers-date-pickers

The accentColor we had been using often didn't work with the dark. It's possible that applications will need to be more assertive about the accent color, there's a lot of variety in the MD spec.